nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -41,10 +38,9 @@
 /** Methods for performing checks related to generic types and nullability. */
 public final class GenericsChecks {
 
-  private static final String NULLABLE_NAME = "org.jspecify.annotations.Nullable";
+  static final String NULLABLE_NAME = "org.jspecify.annotations.Nullable";
 
-  private static final Supplier<Type> NULLABLE_TYPE_SUPPLIER =
-      Suppliers.typeFromString(NULLABLE_NAME);
+  static final Supplier<Type> NULLABLE_TYPE_SUPPLIER = Suppliers.typeFromString(NULLABLE_NAME);
```

Comment:
So this code is about getting a `Type` object for `org.jspecify.annotations.Nullable`.  Some code in this file specifically checks for `@org.jspecify.annotations.Nullable` annotations, and only implements JSpecify behavior if that exact annotation is present.  This was to be conservative; I think it's still an open question as to how we will handle other type use annotations and also declaration annotations (see #853).  I will rename these variables and add some comments to indicate what is going on here.

And, BTW, the reason we code in this manner is to avoid comparing Types by their String representation, see [this Error Prone check](https://errorprone.info/bugpattern/TypeToString).

Reply:
Actually, I'll do those changes in a follow up to keep this PR simple